src/blocks/popup.css

```diff
@@ -19,275 +19,204 @@
   overflow: hidden;
   visibility: visible;
   opacity: 1;
+  transition: visibility 0s, opacity 0.6s;
 }
 
 .popup__container {
   position: relative;
-  top: 57px;
-  left: 0px;
-  background-color: #ffffff;
+  background-color: white;
   width: 100%;
-  max-width: 430px;
-  min-height: 331px;
-  height: auto;
-  border: 1px solid #000;
+  max-width: 358px;
```

Comment:
* The `modals` should be **wide** on `desktop` and **short** only on `mobile`resolutions https://snipboard.io/o9CtN3.jpg . The `width` should be `430px` on desktop https://snipboard.io/khoCPG.jpg 


```css

  max-width: 430px;
```